.nvmrc

```diff
@@ -0,0 +1 @@
+v18
```

Comment:
Can we instead use `.node-version` with `18` instead of `v18`

Reply:
Yeah, of course, we could, I just happen to use `nvm` on my machine, and I think that's the only format it supports. What Node version manager do you use?

Reply:
Yeah need to change those as well, thanks! I use `fnm` https://github.com/Schniz/fnm

Reply:
Looks like there's been an issue open for supporting .node-version for 7 years :smile: https://github.com/nvm-sh/nvm/issues/794